src/Types/ArrayType.php

```diff
@@ -55,14 +55,6 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
         }
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getName()
-    {
-        return Types::ARRAY;
-    }
-
```

Comment:
The only possible BC-break in this PR is removing `getName()` from existing types. It will not work if the type is not registered, or is registered with a different name (which is not supported). If we think this is too risky, it can be reverted without loosing the benefits of having a generic `getName()` implementation.

Obviously, I think it should be fine.

Reply:
I'd revert it. This code itself doesn't cause any problems to be removed in a minor release.